### ISSUE TITLE
DIP

### ISSUE THREAD

**Issue by xiaoyaod** [ryongithub/GatedPromptTuning#1]
how DIP work

**Comment by ryongithub** [comment]
Deep image prior trains a lightweight reconstruction network to restore the original image from fixed random noise. The reconstruction network is trained to generate restored image to be closer to the original image in the representation space of a neural network, rather than in the input space.

Thank you.

**Comment by xiaoyaod** [comment]
Do you have plans to release DIP training code

Whether the input of DIP is the characteristic output of backbone network image

**Comment by ryongithub** [comment]
We utilized the implementation of DIP available at https://github.com/nanxuanzhao/Good_transfer.
(repository for "What Makes Instance Discrimination Good For Transfer Learning?", Zhao et al., ICLR 2021)
In DIP, the input for the reconstruction network is a fixed random noise. But the training loss is computed in the representation space of a neural network.
In our work, we used patch representation from the ViT block for computing loss between the original image and the reconstructed image. 

Thank you
